tomopy/prep/alignment.py

```diff
@@ -443,3 +455,240 @@ def shift_images(prj, sx, sy):
+def remove_slits_aps_1id(imgstacks, slit_box_corners, inclip=(1,10,1,10)):
+    """ 
+    Remove the slits from still images
+
+    Parameters
+    ----------
+    imgstacks         :  np.3darray
+        tomopy images stacks (axis_0 is the oemga direction) 
+    slit_box_corners  :  np.2darray
+        four corners of the slit box 
+    inclip            :  tuple, optional
+        Extra inclipping to avoid clipping artifacts
+
+    Returns 
+    -------
+    np.3darray
+        tomopy images stacks without regions outside slits
+    """
+    xl,xu,yl,yu = calc_slit_box_aps_1id(slit_box_corners, inclip=inclip)
+    return imgstacks[:,yl:yu,xl:xu]
+
+
+def detector_drift_adjust_aps_1id(imgstacks, slit_cnr_ref):
+    """
+    Adjust each still image based on the slit corners and generate report fig
+
+    Parameters
+    ----------
+    imgstacks         :  np.3darray
+        tomopy images stacks (axis_0 is the oemga direction) 
+    slit_cnr_ref      :  np.2darray
+        reference slit corners from white field images
+
+    Returns
+    -------
+    np.3darray
+        adjusted imgstacks
+    np.3darray
+        detected corners on each still image
+    np.3darray
+        transformation matrices used to adjust each image
+    """
+    # -- init slit corners for still images
+    proj_cnrs = []
+
+    # -- init affine transformation matrix
+    img_correct_F = np.ones((imgstacks.shape[0], 3, 3))
+
+    # -- work throgh each image 
+    for n_img in range(imgstacks.shape[0]):
+        # detect the corners
+        proj_cnr = find_slits_corners_aps_1id(imgstacks[n_img,:,:],
+                                              method='quadrant+',
+                                              )
+        proj_cnrs.append(proj_cnr)
+
+        # calcualte the affine transfomration required
+        img_correct_F[n_img,:,:] = calc_affine_transform(proj_cnr, 
+                                                         slit_cnr_ref,
+                                                        )
+
+        # adjust the image
+        if np.linalg.norm(img_correct_F[n_img,0:2,0:2] - np.eye(2)) < 1e-4:
+            # shift only when rotation is small
+            imgstacks[n_img,:,:] = shift(imgstacks[n_img,:,:], 
+                                         img_correct_F[n_img,0:2, 2],
+                                        )  
+        else:
+            # perform full affine transformation
+            # NOTE:
+            #  This is particular slow, need optimization
+            imgstacks[n_img,:,:] = affine_transform(imgstacks[n_img,:,:],                # input image
+                                                    img_correct_F[n_img,0:2,0:2],        # rotation matrix
+                                                    offset=img_correct_F[n_img,0:2,  2], # offset vector
+                                                   )
+    # convert proj_cnrs to np.array
+    proj_cnrs = np.stack(proj_cnrs, axis=0)
+
+    return imgstacks, proj_cnrs, img_correct_F

```

Comment:
vars all lowercase

Reply:
img_correct_F tells people that this is a matrix (M usually represents a matrix/tensor, whereas m represents either a vector or a scalar).  I believe this is better than writing it as img_correct_f or img_correct_matrices.

Reply:
I think it is fine then.